WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java

```diff
@@ -98,14 +98,12 @@ public static boolean isElementCompletelyDisplayed(ViewInteraction element) {
     }
 
     public static void scrollToThenClickOn(Integer elementID) {
-        waitForElementToBeDisplayed(elementID);
```

Comment:
I'm curious about the reason for this removal 🤔 It does save time (theoretically, 0.1 sec), but we are no longer protected from the slow screen load.

Reply:
I asked the same [here](https://github.com/wordpress-mobile/WordPress-Android/pull/17950#discussion_r1115586798), @jostnes feel free to respond to @pachlava here only in order to avoid responding in multiple places. 👍

Reply:
Thank you for the explanation @jostnes! I agree that if we can shorten something, and still have the tests work in the long run the same way, it's justified. My concern was that it was a part of the PR that does something different. However, these words of yours:

> which I only found out from CI testing in this PR.

make it clear 👍

Reply:
Same, thanks for the explantation @jostnes , it is a 👍 from my side too! 🙇 

Suggestion (💡): Might I suggest something that I usually follow myself (for next time). In order for me to avoid explaining something like that here and there, on my PRs or otherwise, I usually prefer to add that to my commit, as part of the its description. This way, the reviewer, or any other engineer that would check this commit/change later on would know why that change happened in full detail, like reading a diary of the author's mind at that point, without needing to have to dig more into that or even ask such questions to the author. I hope that makes sense. This is because a commit like `remove unnecessary wait` cannot help much in these situations, even if you add a PR description like `UI test fixes as the Logout button is not displayed...`.

Btw, sometimes I post these questions of mine just to make the author reply on them and have them re-think it, even re-test it, so they can try and explain it to someone else afterwards and see if that will trigger a change. Then, if the author can explain it, no matter if I agree, of if I can understand the change (🤣),  or even if I don't agree, this will make the author accountable for it and me comfortable to keep things going forward, thus me giving the 👍 on merging, etc.

PS: Please don't mind my rumbling on the above too much, I am just trying to help anyway I can and make us more efficient in terms of commits/code reviews, but I can always be wrong, so please feel free to debate on anything you feel that doesn't click with you, on anything I say! 🙏

Reply:
Thank you for the suggestion @ParaskP7! I guess the change was clear to me and I incorrectly assumed that it would be clear for others as well, I'll be more mindful of this in the future and will include as much information for next time! 🙇